shared/src/commonMain/kotlin/ui/main/children/presence/TimetableItem.kt

```diff
@@ -72,6 +79,32 @@ fun TimetableItem(item: AvailabilityItem, modifier: Modifier, onClick: () -> Uni
         },
         trailingContent = {
             /* TODO: Abel place checkmarks here */
+            Checkbox(
+                checked = checked,
+                onCheckedChange = {
+
+                    // TODO change this, this is temporary
+                    // it means if an admin approved your request you cant unrequest it
```

Comment:
has this check also been implemented on the backend? motto of programming: never trust the client

if someone crafts a custom request that just sets it to absent anyways, the backend will simply listen to it

Reply:
it has now